azure-iot-device/tests/common/pipeline/test_pipeline_stages_mqtt.py

```diff
@@ -298,36 +261,40 @@ def provider_function_throws_base_exception(params, stage, mocker, fake_base_exc
 
 @pytest.mark.parametrize("params", connection_ops + pubsub_ops)
 @pytest.mark.describe(
-    "MQTT Provider pipeline stage _run_op with ops that map directly to provider libraray calls"
+    "Provider - .run_op() -- called with op that maps directly to protocol client library calls"
 )
 class TestMQTTProviderBasicFunctionality(object):
-    @pytest.mark.it("calls the appropriate function on the provider library")
+    @pytest.mark.it("Calls the appropriate function on the protocol client library")
```

Comment:
>Calls the appropriate function on the protocol client library [](start = 21, length = 61)

Not sure about a lot of the phrasing in this test class.

- Not really calling a function, but calling a method
- It's not really calling the protocol client library. That would be Paho, right?. This is calling into the MQTTProvider class. I don't know what to call that though tbh.